util/execdetails/execdetails.go

```diff
@@ -52,3 +54,55 @@ func (d ExecDetails) String() string {
 	}
 	return strings.Join(parts, " ")
 }
+
+// ExecStats collects executors's execution info.
+type ExecStats map[string]*ExecStat
+
+// ExecStat collects one executor's execution info.
+type ExecStat struct {
```

Comment:
1. How about: `s/ExecStat/RuntimeStats/`
2. Please add comments for each struct field.